src/game/shell.c

```diff
@@ -41,12 +41,12 @@ static char *Shell_GetScreenshotName(void)
 {
     // Get level title of unknown length
     char level_title[100];
-    snprintf(
+    size_t out_size = snprintf(
```

Comment:
It's possible for snprintf to be negative so I'm not sure size_t will work? Isn't it always unsigned?

https://cplusplus.com/reference/cstdio/snprintf/

Reply:
I suspect that `out_size` was made as `size_t` because it was used as parameter to `Memory_Alloc()`.

Reply:
Can we we make the `out_size` and `int` and error check if the return value is not -1 for both `snprintf` calls? Then maybe cast to `size_t` in the `Memory_Alloc` call?

Also this might be an @rr- question, but maybe we can update this PR to solve #445 while we are messing with this code?

Reply:
IMO unrelated change, I'd touch that in a separate PR / commit.